src/ResourceManager/Network/Commands.Network/Microsoft.Azure.Commands.Network.format.ps1xml

```diff
@@ -2,6 +2,32 @@
 <Configuration>
   <ViewDefinitions>
     <View>
+      <Name>Microsoft.Azure.Commands.Network.Models.PSUsage</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.Commands.Network.Models.PSUsage</TypeName>
+      </ViewSelectedBy>
+      <ListControl>
```

Comment:
The ddefault here should be a table view - with only 3 columns, this makes the display much easier.  Users can always use the list view by piping through ```fl```